packages/cozy-stack-client/src/TriggerCollection.js

```diff
@@ -171,8 +171,38 @@ class TriggerCollection extends DocumentCollection {
     }
   }
 
-  async update() {
-    throw new Error('update() method is not available for triggers')
+  /**
+   * Updates a Trigger document. Only updatable attributes plus _id are allowed.
+   *
+   * @param  {object}  trigger Trigger's attributes to update + id
+   * @returns {object}  Stack response, containing resulting trigger document under `data` attribute.
+   */
+  async update(trigger) {
+    for (const key in trigger) {
+      if (!['_id', 'arguments', 'message'].includes(key)) {
+        throw new Error(
+          `TriggerCollection.update only works for 'arguments', and 'message' attributes.`
+        )
+      }
+    }
```

Comment:
you should not use `update`. You should use `save` in order to not bypass cozy-client links.